### ISSUE TITLE
PointCloudMulti in dataset

### ISSUE THREAD

**Issue by rmraaron** [Sentient07/IFMatch#6]
Hi Ramana, 

Thanks for your work. It's really cool. 

I have a question about the dataset. I noticed the `len` of `PointCloud_with_FreePoints` is `max_points`, i.e. 200000. When shuffling items in `PointCloudMulti`, does this mean that for each instance, it is obtained for 200000 times but with different random indices for on_surface points, free points, etc? 

Looking forward to hearing from you.

**Comment by Sentient07** [comment]
Hello,

[Here](https://github.com/Sentient07/IFMatch/blob/master/dataset.py#L67), the length of dataloader is `280000 / on_surface_points`. And each batch contains in total `on_surface_points*2` points consisting of on and off-surface points. Hope that helps.

**Comment by rmraaron** [comment]
> Hello,
> 
> [Here](https://github.com/Sentient07/IFMatch/blob/master/dataset.py#L67), the length of dataloader is `280000 / on_surface_points`. And each batch contains in total `on_surface_points*2` points consisting of on and off-surface points. Hope that helps.

Thank you for your quick reply. Can I ask how long it took to train FAUST for 36 epochs? Thanks!

**Comment by Sentient07** [comment]
I'm not sure I remember, but I hope it shouldn't be more than 2hrs for 80 shapes on A100 GPUs. I'm closing this issue but please feel free to reopen
